scripts/gitlab/build_and_test.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
```

Comment:
Instead of introducing a new top-level directory, can this be put under `tests`?

i.e. `scripts/gitlab/*` -> `tests/gitlab/*`

Reply:
Sure! that's an artifact of pervious introduction of uberenv...